subcommittees/mentoring/minutes/minutes-2017-08-28.md

```diff
@@ -0,0 +1,276 @@
+
+- Survey:
+  - What will we do with results?
+  - Plan to share through casual methods (social media, slack, AMY email lists (discussion list), Toby to write a blog post)
+  - Encourage people to also sign up for calendar and Slack/social media
+
+#### Discussion Coordinator - Volunteer position
+
+- Some interest from Paula on 1st call - Paula and Jamie working on onboarding document for this role (link to be posted)
+
+#### Getting community more engaged/participating in instructor discussion sessions
+
+- Some kind of credit system? Rewarding those who host/co-host (i.e. through a monthly drawing)
+
+### Main Agenda Items
+
+#### Updates from liaisons
+
+- __Training Committee (Erin):__
+  - "We have been doing a number of open instructor training events, reaching many new instructors in countries without a large Carpentries presence. So it would be good to let the mentoring committee know that they'll be seeing folks with different backgrounds coming through the discussion sessions."
+  - "We're also planning a new release of the instructor training materials in the next couple of weeks that are significantly updated. I don't know if this will impact anything in the discussions, but trainees might start using some new vocab that older instructors aren't as familiar with."
+
+###### First Meeting
+
+- MK: Every session that I host, I get at least one email asking to join. Many more checkouts than debriefs/pre-workshop. I don't think that this is good for the sessions. Already hard, but increasing the number of sessions won't (necessarily) increase the number of pre-/debriefs.
+- CK: Need to have a discussion amongst the instructor trainers. I think that limiting number of slots has helped.
+- JH: Erin is working on a summary of the changes in this new release
+
+- __Steering Committee (Christina):__
+  - The Software and Data Carpentry re-organization is moving along. See the latest blog post announcement here: https://software-carpentry.org/blog/2017/08/mergermotions.html  Relevant information for this community: 
+    - In the future, sub-committees will have a staff point person, so after the merger, someone from the new Carpentries organization will be helping with the mentoring committee
+    - The  mentoring committee is something that applies across carpentries, so we will be helping move out of the swcarpentry/board repo into a carpentries/mentoring repo soon
+  - More discussion is happening about CarpentryCon, people in this committee might be interested in checking it out.
+    - http://pad.software-carpentry.org/2018carpentrycontaskforce
+
+###### First Meeting
+
+- MK: now the CarpentryCon committee knows more what they want - more of an unconference, keynote, interactive sessions - two days with an extra day for community-organized events - but two days would be the core - more venue requirements set out by group
+- MK: links to carpentrycon documents: https://drive.google.com/open?id=0B70o76-4Jmd0NzRteURZRWRhZGc
```

Comment:
this link does not work and I cannot get the shareable link to this drive, I'd remove it, at least until I know why it does not work

Reply:
I think this is the right link - https://drive.google.com/drive/u/1/folders/0B2Mb22lbj2tjMFBiVE5oQ3ctZHc

but I can't access without permission (which Ive now requested), so I propose that I instead encourage people in the minutes to attend the ext CarpentryCon call on 7th Sep instead.